test/Microsoft.AspNet.Mvc.Xml.Test/FlushReportingStream.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading;
+using Moq;
+
+namespace Microsoft.AspNet.Mvc.Core
```

Comment:
So the namespace for all tests should be: `Microsoft.AspNet.Mvc` ?

Scanning through the files in Microsoft.AspNet.Mvc.Core.Test, it looks like the namespaces are all over the place:
```csharp
Microsoft.AspNet.Mvc
Microsoft.AspNet.Mvc.Core
Microsoft.AspNet.Mvc.Core.Test
Microsoft.AspNet.Mvc.OptionDescriptors
```